addons/project/static/src/components/project_right_side_panel/components/project_right_side_panel_section.xml

```diff
@@ -2,14 +2,14 @@
 <templates id="template" xml:space="preserve">
 
     <t t-name="project.ProjectRightSidePanelSection">
-        <div class="o_rightpanel_section py-0" t-att-name="props.name" t-if="props.show">
-            <div class="o_rightpanel_header d-flex align-items-center justify-content-between py-2 border-bottom" t-att-class="props.headerClassName" t-if="props.header">
-                <div class="o_rightpanel_title me-auto" t-if="props.slots.title" t-attf-class="{{ env.isSmall ? 'd-flex flex-row-reverse align-items-center' : '' }}">
-                    <h3 class="m-0 lh-lg ps-1"><t t-slot="title"/></h3>
+        <div class="o_rightpanel_section" t-att-name="props.name" t-if="props.show">
+            <div class="o_rightpanel_header d-flex align-items-center justify-content-between gap-1 pe-3" t-att-class="props.headerClassName" t-if="props.header">
```

Comment:
The o_rightpanel_header doesn't seem to have any style/js associated, could you check if it's still necessary?